webdriver-spec.html

```diff
@@ -128,6 +128,7 @@
    <!-- Fire an event --> <li><dfn data-lt="fire|fires|fired|event fires|event fired"><a href=https://dom.spec.whatwg.org/#concept-event-fire>Fire an event</a></dfn>
    <!-- Get an attribute by name --> <li><dfn data-lt="getting an attribute by name"><a href=https://dom.spec.whatwg.org/#concept-element-attributes-get-by-name>Get an attribute by name</a></dfn>
    <!-- getAttribute --> <li><dfn><a href=https://dom.spec.whatwg.org/#dom-element-getattribute>getAttribute</a></dfn>
+   <!-- getElementsByTagName --> <li><dfn><a href="https://dom.spec.whatwg.org/#dom-element-getelementsbytagname">getElementsByTagName</a></dfn>
```

Comment:
can you wrap `getElementsByTagName` in `<code>` tag e.g. `<code>getElementsByTagName</code>` and then update the text below to wrap it there too.